index.html

```diff
@@ -11,34 +11,217 @@
+                                  <li><a href="#" class="boot-btn">bootstrap</a></li>
+                                  <li><a href="#" class="ruby-btn">Ruby</a></li>
+                                </ul>
+                            </div>
+                            <ul>
+                                <a href="#"><button class="some-pro-text hover-effect">See Project</button></a>
+                            </ul>  
+                        </div>
+                        <div class="all-cards">            
+                            <div class="no-btn">
+                                <h1>Profesional Art Printing Data</h1>
+                                <p>
+                                   A daily selection of privately personalized reads; no accounts or sign-ups required. 
+                                   has been the industry's standard
+                                </p>
+                                <ul class="cards-lists">
+                                  <li><a href="#"class="html-btn">html</a></li>
+                                  <li><a href="#" class="boot-btn">bootstrap</a></li>
+                                  <li><a href="#" class="ruby-btn">Ruby</a></li>
+                                </ul>
+                            </div>
+                            <ul>
+                                <a href="#"><button class="some-pro-text hover-effect">See Project</button></a>
+                            </ul>  
+                        </div>
+                    </div>  
+                </div>
+            </div>
+        </div>
+        <section class="about-container">
+            <div class="first-about-container">
+                <h1 class="about-header">About me</h1>
+                <p class="about-para">Hello I’m a software developer! I can help you build a product , 
+                    feature or website Look through some of my work and experience! 
+                    If you like what you see and have a project you need coded, don’t hestiate to contact me.
+                </p>
+                <a href="#"><button class="about-btn-text hover-effect">Get My Resume</button></a>
+                <img class="about-illu" src="./img/about/illustration-about-me-mobile.svg" alt="illustration">
+            </div>
+            <img class="about-divider" src="./img/about/Vector 4.svg" alt="">
+            <div class="skills-container">
+                <div class="language">
+                    <ul class="language-header">
+                        <li><img src="./img/about/Rectangle 55.svg" alt="Rectangle"></li>
+                        <li class="language-h1">Language</li>
+                    </ul>
+                    <ul class="main-language">
+                        <li class="skill-js">JavaScript</li>
+                        <li class="skill-rub">Ruby</li>
+                        <li class="skill-html">Html</li>
+                        <li class="skill-css">Css</li>
```

Comment:
Please consider using the same font as used in the template here in all the list items to make it like the design.

## Solution:-

Add the following CSS rule for the list items:-

```

    font-family: 'Inter', sans-serif;

```